runtime/src/lib.rs

```diff
@@ -456,6 +459,12 @@ parameter_types! {
 	pub MinSolutionScoreBump: Perbill = Perbill::from_rational_approximation(5u32, 10_000);
 }
 
+type SlashCancelOrigin = EnsureOneOf<
+	AccountId,
+	EnsureRoot<AccountId>,
+	pallet_collective::EnsureProportionAtLeast<_3, _4, AccountId, CouncilCollective>,
```

Comment:
as there will be only once council member, EnsureMember makes a bit more sense

Reply:
I have to fix babe/grandpa equivocations at first to slashing start to works. I can make it in 2 issues/PRs if we want

Reply:
Ok so the slashing wasn't working in the first place, got it. Probably no need to split it at this point.